client/gefyra/api/bridge.py

```diff
@@ -94,14 +94,24 @@ def bridge(
         )
         return False
 
-    workload_type, workload_name, container_name = target.split("/", 2)
+    try:
+        _bits = list(filter(None, target.split("/")))
+        workload_type, workload_name = _bits[0:2]
+        container_name = _bits[2] if _bits[2:] else None
+    except IndexError:
+        logger.error(
```

Comment:
I'd say this has to be a RuntimeError, as the Gefyra client API should not know about CLI switches.

Reply:
Changed to `RuntimeError`